src/Farmer/Builders/Builders.PostgreSQL.fs

```diff
@@ -405,18 +506,25 @@ type PostgreSQLBuilder() =
         }
 
     /// Adds a custom vnet rule given a name and a virtualNetworkSubnetId.
+    [<Obsolete(SingleServerObsoleteMessage)>]
     [<CustomOperation "add_vnet_rule">]
```

Comment:
Ah so this is now just a part of the Server itself rather than a separate virtualnetwork resource. I'll check this and update if possible. Worse case we merge like this and then add VNet support at a later date?

Reply:
@ninjarobot do you think that in the Flexible Server world, `delegatedSubnetResourceId` maps to what was `virtualNetworkSubnetId`?

Reply:
Yes, we probablly need to think about that more. New builder - yes, we probably would then remove / replace the current one completely (or at least delete the obsolete methods + types).